plugins/blockvolume/blockprovider/blockvolume_provider.go

```diff
@@ -20,6 +20,7 @@ var (
 type Provider interface {
 	CreateBlockVolume(name string, size uint64, hostVolume string, options ...BlockVolOption) (BlockVolume, error)
 	DeleteBlockVolume(name string, options ...BlockVolOption) error
+	GetAndDeleteBlockVolume(name string, options ...BlockVolOption) (BlockVolume, error)
```

Comment:
why we need this?

Reply:
In DeleteVolume() we need the information of the host volume to prune the bhv. DeleteBlockVolume() doesn't return the host vol info, hence the func GetAndDeleteBlockVolume()